packages/fauna/README.md

```diff
@@ -1,40 +1,67 @@
-# Fauna Adapter
-
-### Initial Setup
-
-```javascript
-CreateCollection({ name: 'accounts' })
-CreateCollection({ name: 'sessions' })
-CreateCollection({ name: 'users' })
-CreateCollection({ name: 'verification_requests' })
-CreateIndex({
-  name: 'account_by_provider_account_id',
-  source: Collection('accounts'),
-  unique: true,
-  terms: [
-    { field: ['data', 'providerId'] },
-    { field: ['data', 'providerAccountId'] },
-  ],
-})
-CreateIndex({
-  name: 'session_by_token',
-  source: Collection('sessions'),
-  unique: true,
-  terms: [{ field: ['data', 'sessionToken'] }],
-})
-CreateIndex({
-  name: 'user_by_email',
-  source: Collection('users'),
-  unique: true,
-  terms: [{ field: ['data', 'email'] }],
+<p align="center">
+   <br/>
+   <a href="https://next-auth.js.org" target="_blank"><img width="150px" src="https://next-auth.js.org/img/logo/logo-sm.png" /></a>
+   <h3 align="center"><b>Fauna Adapter</b> - NextAuth.js</h3>
+   <p align="center">
+   Open Source. Full Stack. Own Your Data.
+   </p>
+   <p align="center" style="align: center;">
+      <img src="https://github.com/nextauthjs/adapters/workflows/Build%20Test/badge.svg" alt="Build Test" />
+      <img src="https://img.shields.io/bundlephobia/minzip/@next-auth/fauna-adapter" alt="Bundle Size"/>
+      <img src="https://img.shields.io/github/v/release/nextauthjs/adapters?include_prereleases" alt="Github Release" />
+   </p>
+</p>
+
+## Overview
+
+This is the Fauna Adapter for [`next-auth`](https://next-auth.js.org). This package can only be used in conjunction with the primary `next-auth` package. It is not a standalone package.
+
+You can find the Fauna schema and seed information in the docs at [next-auth.js.org/schemas/fauna](https://next-auth.js.org/schemas/fauna).
```

Comment:
This concerns the other adapters as well, but I actually thought it would be better the other way around. So have the schema in this repo as a file, and link to it from the docs page. Schemas are very useful for the tests, so it is good to have them next to the adapters' source code.

Reply:
Okay yeah no problem, ill add the Fauna setup code back in in its own file as well :+1: